docs/DeveloperGuide.md

```diff
@@ -428,6 +464,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Add a Schedule**
+
+**MSS**
+
+1. FitEgo shows a list of Clients and list of Sessions
+
+2. User requests to add a specific Schedule between a specified Client from Client List and Session from Session List
+
+3. FitEgo adds the Schedule
+
+   Use case ends.
+
+**Extensions**
+
+- 2a. The Schedule to be added is overlapping with another Schedule
+
+  - 2a1. FitEgo shows an error message
+
+    Use case resumes at step 2.
```

Comment:
Want to update to this? 
![image](https://user-images.githubusercontent.com/14960144/96665709-12fd4400-1388-11eb-9cf6-9277f4eae905.png)